### PR TITLE
feat: Modal component Responsive Web Develop

### PR DESCRIPTION
**Pull request by juntaepark**

### CODE REVIEW

File: packages/web/src/components/common/notification-list/NotificationList.styles.ts [hunk 1]
```diff
@@ -2,11 +2,18 @@ import { fonts } from "@constants/font.constant";
...
+export interface NotificationProps {
+  width: CSSProperties["width"];
+}
+
+export const NotificationListWrapper = styled.div<NotificationProps>`
   position: absolute;
   top: 53px;
-  right: -150px;
+  right: ${({ width }) => {
+    return width && width > 1680 ? "-150px" : "0px";
```
Comment (jinoosss) [inline packages/web/src/components/common/notification-list/NotificationList.styles.ts:15]: The type of `width` is either string or number. 
If it is a string, it cannot be compared to 1680.

File: packages/web/src/components/common/wallet-connector-menu/WalletConnectorMenu.styles.ts [hunk 2]
```diff
@@ -3,11 +3,19 @@ import styled from "@emotion/styled";
...
+export interface WalletMenuProps {
+  width: CSSProperties["width"];
+}
+
+export const WalletConnectorMenuWrapper = styled.div<WalletMenuProps>`
   position: absolute;
   width: 280px;
   top: 54px;
+  right: ${({ width }) => {
+    return width && width < 1521 && "0px";
```
Comment (jinoosss) [inline packages/web/src/components/common/wallet-connector-menu/WalletConnectorMenu.styles.ts:17]: The type of width is either string or number. If it is a string, it cannot be compared to 1521.

File: packages/web/src/components/common/notification-list/NotificationList.styles.ts (hunk 1 above)
Comment (juntaepark) [reply packages/web/src/components/common/notification-list/NotificationList.styles.ts:15]: correct.

File: packages/web/src/components/common/wallet-connector-menu/WalletConnectorMenu.styles.ts (hunk 2 above)
Comment (juntaepark) [reply packages/web/src/components/common/wallet-connector-menu/WalletConnectorMenu.styles.ts:17]: correct.

**Review by jinoosss — APPROVED** [review]
Approve !

**Pull request merged**